evm/src/cpu/kernel/asm/hash/blake2b/permutations.asm

```diff
@@ -0,0 +1,194 @@
+global permutation_0_constants:
+    BYTES 0
```

Comment:
nit: maybe we should have default number of bytes per line? before you use 4 but here it's 1.

Reply:
Ohh good point — I vaguely thought they had to be on separate lines to be in separate memory words, but turns out it works fine if it's 4 per line